website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -13,6 +13,22 @@ within the same site as the gateway.
 - Docker Engine (for docker-based installs)
 - Systemd (for systemd-based installs)
 
+## Firewall considerations
+
+Gateways implement the industry-standard
+[STUN](https://datatracker.ietf.org/doc/html/rfc8489) and
+[TURN](https://www.rfc-editor.org/rfc/rfc8155.html) protocols to securely
+perform NAT traversal and firewall hole-punching. No ingress firewall ports are
+required for Gateways to function.
+
+If the network in which your Gateway is deployed performs egress filtering,
+you'll need to make sure the following outbound traffic is allowed:
+
+| IP Address           | Port(s) | Protocol | Purpose                  |
+| -------------------- | ------- | -------- | ------------------------ |
+| `34.102.202.25`      | `443`   | `TCP`    | Control Plane API (IPv4) |
+| `2600:1901:0:620b::` | `443`   | `TCP`    | Control Plane API (IPv6) |
+
```

Comment:
TODO: Add Relays

Reply:
I would avoid giving static IPs for relays for now, it's a ton of maintenance because we are still figuring out our scaling and deployment regions for them

Reply:
Can we give a range at least? I think users are expecting a list of IPs to whitelist for their firewall rules.

Reply:
I don't know if we can allocate an entire range of IPs (probably means we will have to pay for them), so the range would be like entire GCP IPs range which doesn't make any sense to whitelist

Reply:
Yeah, we can buy IPs one by one for 7$ per one monthly (3$ if it's used) or bring our own IPs (50$+ per IP).

Reply:
Thanks for the info, yeah, I'll just give port and proto requirements right now and update this to clarify when we are able to lock them down to static IPs